### ISSUE TITLE
New simple Features

### ISSUE THREAD

**Issue by MaskBoyz** [charlesbel/Microsoft-Rewards-Farmer#223]
I think its will be helpful if any accounts get suspended or have issue like unusual activities its its will print out like account suspended or unusual activities and and log file so we can check in the log files if all the points have been collected or there was some problem or the account got suspended. Logs will be best helpful cause then we dont have log in every account to check and whenever we run the script its will check the log if this account has completed successfully then its skip the account and log in to next account

**Comment by malesci** [comment]
I agree! It will be also great to have logs on Discord, Telegram or Google Spreadsheet row.
thanks

**Comment by charlesbel** [comment]
> it would also be appreciated if the bot could do the unusual activity automatically using some fakemail i've tested and https://temp-mail.org/ seems to work when selected email as a recovery method. and the logs on discord suggested above aswell.

This is not recommended if you want to avoid a permanent ban

**Comment by charlesbel** [comment]
> I think its will be helpful if any accounts get suspended or have issue like unusual activities its its will print out like account suspended or unusual activities and and log file so we can check in the log files if all the points have been collected or there was some problem or the account got suspended. Logs will be best helpful cause then we dont have log in every account to check and whenever we run the script its will check the log if this account has completed successfully then its skip the account and log in to next account 

Will look at the fork that already implement it to add it here 😉

**Comment by MaskBoyz** [comment]
bro i dont really understand how the command will be with telegram bot token id and chat id

**Comment by charlesbel** [comment]
> bro i dont really understand how the command will be with telegram bot token id and chat id
> 

-t bot-id chat-id
As simple as that

**Comment by MaskBoyz** [comment]
> > bro i dont really understand how the command will be with telegram bot token id and chat id
> 
> -t bot-id chat-id As simple as that

I tried but not receiving any message from bot  I will try again later

**Comment by charlesbel** [comment]
> > > bro i dont really understand how the command will be with telegram bot token id and chat id
> > 
> > -t bot-id chat-id As simple as that
> 
> I tried but not receiving any message from bot  I will try again later 

Please check #276 comments, they report it as working
